src/Payone/RequestParameter/Builder/SecuredInvoice/AuthorizeRequestParameterBuilder.php

```diff
@@ -82,4 +85,34 @@ protected function getOrder(string $orderId, Context $context): OrderEntity
 
         return $order;
     }
+
+    protected function applyB2bParameters(OrderEntity $order, array &$parameters): void
+    {
+        /** @var OrderAddressCollection $addresses */
+        $addresses = $order->getAddresses();
+        /** @var OrderAddressEntity $billingAddress */
+        $billingAddress = $addresses->get($order->getBillingAddressId());
```

Comment:
will be never `null`, but could be null. Please adjust the following code, that it will not break with a fatal error.
Please also adjust the phpdoc.

Reply:
ah sorry, you already checked for a null value. But the doc-block does not match the type `OrderAddressEntity|null`

Reply:
I have removed the phpdoc and improved the code.